api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -98,6 +99,14 @@ from \`${BQ_PROJECT}.${BQ_DATASET}.person\`
 where floor((2018 - year_of_birth)/10) >=3
 group by stratum_2"
 
+# 7 Gender identity count
```

Comment:
The content here definitely feels like it's grown well beyond the limits of a shell script. There's a pretty high degree of duplication in this file and it is quite difficult to review.

Not for this PR, but possibly a future refactoring ticket.

Reply:
I was thinking about that too. Has been adding up things lately. Will take care of this in next pr.